### PR TITLE
Made mobile Visitor Table style break over lines when needed

### PR DESCRIPTION
**Pull request by vieee**
- Added functionality to Visitor table so that is breaks in multiple lines when table queries extend line limit

### CODE REVIEW

**Comment by vieee** [comment]
Hi @KonradLinkowski, this setup auto-scales on table rows of higher word length, is this fine..?

**Review by KonradLinkowski — CHANGES_REQUESTED** [review]
Looks cool, but why did you remove yarn. lock?

**Comment by vieee** [comment]
Oh, by mistake, my bad.. Will add it..

File: public/assets/public.css [hunk 1]
```diff
@@ -137,7 +144,8 @@ only screen and (max-width: 760px),
 		border: none;
 		border-bottom: 1px solid #eee; 
 		position: relative;
-		padding-left: 50%; 
+    padding-left: 50%; 
+    word-wrap:break-word;
```
Comment (KonradLinkowski) [inline public/assets/public.css:148]: Some tabulation issue?

File: public/index.html [hunk 2]
```diff
@@ -48,7 +48,9 @@ <h2 class="panel__title">Visitors Table</h2>
             <th class="visitors__hcell">Longitude</th>
             <th class="visitors__hcell">Date</th>
           </thead>
-          <tbody class="visitors__body"></tbody>
+          <tbody class="visitors__body">
+
```
Comment (KonradLinkowski) [inline public/index.html:52]: ?

**Review by KonradLinkowski — CHANGES_REQUESTED** [review]
I could approve this, but it would be cool if the css was formatted as before.

File: public/index.html (hunk 2 above)
Comment (vieee) [reply public/index.html:52]: Didn't see this yesterday, sorry.. 🙈
There is no issue in tables..

**Pull request merged**